roles/webview/templates/var/www/webview/scripts/settings.js

```diff
@@ -18,48 +22,33 @@
         host: location.hostname
       },
 
-      // Prefix to prepend to page titles
-      titleSuffix: ' - OpenStax CNX',
-
-      // Google Analytics tracking ID
-      analyticsID: 'UA-7903479-1',
-
-      // Supported languages
-      languages: languages,
-
       // Legacy URL
       // URLs are concatenated using the following logic: location.protocol + '//' + legacy + '/' + view.url
       //   Example: 'http:' + '//' + 'cnx.org' + '/' + 'contents'
       // Do not include the protocol or a trailing slash
       legacy: '{{ zope_domain }}',
 
       // Webmaster E-mail address
-      webmaster: 'cnx@cnx.org',
+      webmaster: '{{ webmaster_email|default("support@openstax.org") }}',
 
       // Content shortcodes
-      shortcodes: {
-      },
+      shortcodes: {{ content_shortcodes|default({})|to_json }},
 
       accountProfile: 'https://{{ accounts_domain }}/profile',
       cnxSupport: 'http://openstax.force.com/support?l=en_US&c=Products%3ACNX',
-      terpUrl: function (itemCode) {
-        return 'https://openstaxtutor.org/terp/' + itemCode + '/quiz_start';
-      },
 
       exerciseUrl: function (itemCode) {
         return 'https://{{ exercises_domain|default("exercises-qa.openstax.org") }}/api/exercises?q=tag:' + itemCode;
       },
 
-      defaultLicense: {
-        code: 'by'
-      },
-
       conceptCoach: {
         uuids: {{ concept_coach_webview_settings|default({})|to_json }},
-        url: 'https://{{ tutor_domain|default("tutor-qa.openstax.org") }}'
+        url: 'https://{{ tutor_domain|default("tutor-qa.openstax.org") }}',
```

Comment:
Please remove the defaults here. tutor_domain is a required variable. We don't want production to accidentally point at the qa environment.